packages/js/email-editor/src/components/sidebar/style.scss

```diff
@@ -8,17 +8,3 @@
 	// resetting margin fits the notice to the full column width
 	margin: 0;
 }
-
-// Disable focus outline for buttons in the sidebar
-.edit-post-sidebar {
-	.components-button:focus:not(:disabled) {
-		box-shadow: none;
-		outline: none;
-	}
-
-	// Because we use complementary area component from the WP interface package the is visible an unwanted close icon.
-	// This style hides it.
-	.interface-complementary-area-header__small {
-		display: none;
-	}
-}
```

Comment:
We can remove this as well. https://github.com/woocommerce/woocommerce/blob/173bae563b0e0741f3c1cb63760696a7715be2a1/packages/js/email-editor/src/components/styles-sidebar/style.scss#L55-L59